plugins/discourse-narrative-bot/spec/discourse_narrative_bot/track_selector_spec.rb

```diff
@@ -63,7 +63,7 @@
 
       let(:post) { Fabricate(:post, topic: topic, user: user) }
 
-      context 'during a tutorial track' do
+      context 'when during a tutorial track' do
```

Comment:
Not sure how to name it better, but "when" and "during" don't play well together. Maybe "while in a tutorial track"?